app/components/Footer.tsx

```diff
@@ -1,10 +1,165 @@
+import { APP_NAME, Categories, DISCORD_URL, X_URL } from '@/constants';
+import Link from 'next/link';
+
 export const Footer = () => (
-  <div className="flex items-center justify-center space-x-3 px-6 pt-32">
-    <p>
-      built by{' '}
-      <a href="https://swapr.eth.limo" className="text-text-primary-main">
-        Swapr.eth
+  <div className="mt-20 border-t border-surface-surface-2 bg-surface-surface-0 px-6 md:px-10 lg:px-20 xl:px-40">
+    <div className="flex flex-col-reverse items-start justify-between py-10 md:flex-row">
+      <div className="w-full space-y-12 md:w-fit">
+        <div className="hidden space-y-2 md:block">
+          <Logo />
+          <p>Where prediction markets and AI agents meet</p>
+        </div>
+        <PoweredByPartners />
+      </div>
+      <div className="flex space-x-10">
+        <div>
+          <p className="mb-2 text-lg font-bold">Markets</p>
+          <ul className="space-y-1">
+            {Object.values(Categories).map(category => (
+              <li className="text-md font-medium capitalize" key={category}>
+                <a href={`/?c=${category}`}>
+                  <p className="hover:text-text-primary-main">{category}</p>
+                </a>
+              </li>
+            ))}
+          </ul>
+        </div>
+        <div>
+          <p className="mb-2 text-lg font-bold">Socials</p>
+          <ul className="space-y-1">
+            <li className="text-md capitalize">
+              <a href={X_URL} target="_blank">
+                <p className="hover:text-text-primary-main">X</p>
+              </a>
+            </li>
+            <li className="text-md capitalize">
+              <a href={DISCORD_URL} target="_blank">
+                <p className="hover:text-text-primary-main">Discord</p>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <MadeBySwapr />
+  </div>
+);
+
+const PoweredByPartners = () => (
+  <div className="flex flex-col space-y-2 rounded-8 bg-surface-surface-1 p-3 md:flex-row md:items-center md:space-x-2 md:space-y-0">
+    <p className="mb-2 text-nowrap md:mb-0 md:mr-2">Powered by</p>
+    <div className="grid grid-cols-2 gap-2 md:flex">
```

Comment:
Ahhh love this change, I like this better than the single column <3, great work!